src/lib/ContractTemplates/ERC721Mintable.js

```diff
@@ -36,7 +49,7 @@ export default class ERC721Mintable {
    * @notice Warning: This method will consume gas (4000000 gas estimated)
    * @returns void
    */
-  async deploy({ name, symbol, contractURI }) {
+  async deploy({ name, symbol, contractURI, gas = null }) {
```

Comment:
why not call that `options = {}` so that we have a placeholder for other options + allowing user to extend the options of the etherjs call?

Reply:
gets a bit convoluted with our gas limit that we have to set 😬 
can be updated in a follow up PR if we think it works better